test/spec/shield.js

```diff
@@ -27,8 +27,8 @@ const handler = shieldRenderer.getStyleImageMissingHandler();
 
 handler({ id: "shield\nBAB\n5\n\n" });
 handler({ id: "shield\nUS:RI\n\n\n" });
-handler({ id: "shield\nUS:RI\nABC123\n\n" });
-handler({ id: "shield\nUS:RI\nEquator\n\n" });
+handler({ id: "shield\nUS:RI\nABC1234\n\n" });
+handler({ id: "shield\nUS:RI\nLatitude\n\n" });
```

Comment:
Is this change intended?

Reply:
Yes; this is related to changing the max ref length to 7.